book/installation.rst

```diff
@@ -261,6 +259,11 @@ If there are any issues, correct them now before moving on.
 
     If this doesn't work, try adding ``-n`` option.
 
+    .. note::
+
+        setfacl isn't available on NFS mount points. However, setting cache and logs over NFS is strongly not recommended for performance.
```

Comment:
Can you please wrap the paragraph after the first word that crosses the 72nd character?